src/build/installation/installation.md

```diff
@@ -325,18 +354,24 @@ Although this command is used to run a function, it also builds your program.
 ## Deploy
 
 ```
-snarkos developer deploy {$PROGRAM_NAME} --private-key {$PRIVATE_KEY} --query "https://api.explorer.aleo.org/v1" --path ".build/" --broadcast "https://api.explorer.aleo.org/v1/testnet3/transaction/broadcast" --priority-fee 0
+# If you have the private key present in your .env file
+leo deploy
+
+#Otherwise
+leo deploy --private-key <your_pivate_key>
 ```
 If successful, it should look something like
-![Deploy confirmation](./images/deploy-confirmation.png)
-
+![Deploy Confirmation](./images/deployTokenUpdated.png)
 The hash at the bottom of the image represents the transaction hash, which you can look up the transaction details on an [explorer](https://explorer.aleo.org)
 
 ## Execute
+
+You can now easily execute programs on-chain, even if they don't match the local directory. Make sure to have a funded private key in .env or specify one using --private-key.
 ```
-snarkos developer execute {$PROGRAM_NAME} {$TRANSITION_NAME} {$INPUT_ARGUMENTS} --private-key {$PRIVATE_KEY} --query "https://api.explorer.aleo.org/v1" --broadcast "https://api.explorer.aleo.org/v1/testnet3/transaction/broadcast"
+leo execute --program {PROGRAM_NAME} --broadcast {transition/function name} <ADDRESS> 500u64
```

Comment:
Try to be consistent with the command parameters
leo execute --program <program_name> --broadcast <transition_function_name> <function_inputs_separated_by_space>

Reply:
Updated